.github/workflows/release.yml

```diff
@@ -337,90 +350,180 @@ jobs:
 
   ####### Prepare the release draft #######
 
+  build-srtool-runtimes:
+    runs-on: self-hosted
+    needs: ["set-tags"]
+    if: needs.set-tags.outputs.is_release == 'true'
+    strategy:
+      matrix:
+        runtime: ["moonbase", "moonshadow", "moonriver", "moonbeam"]
+    container:
+      image: paritytech/srtool:nightly-2021-03-15
+      volumes:
+        - ${{ github.workspace }}:/build
+      env:
+        PACKAGE: ${{ matrix.runtime }}-runtime
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache target dir
+        uses: actions/cache@v2
+        with:
+          path: "${{ github.workspace }}/runtime/${{ matrix.runtime }}/target"
+          key: srtool-target-${{ matrix.runtime }}-${{ github.sha }}
+          restore-keys: |
+            srtool-target-${{ matrix.runtime }}-
+            srtool-target-
+      - name: Build ${{ matrix.runtime }} runtime
+        id: build-runtime
+        shell: bash
+        env:
+          srtool_output_filename: ${{ matrix.runtime }}_srtool_output.json
+        run: |
+          cd /build
+          pwd
+          ls -la
```

Comment:
Some debugging lines.

Reply:
Actually, if they might be useful again, I'm fine leaving them in.

Reply:
removed